web-client/src/views/JudgeActivityReport/JudgeActivityReport.tsx

```diff
@@ -339,7 +339,12 @@ export const JudgeActivityReport = connect(
                 </select>
               </div>
 
-              <div className="grid-col-auto display-flex flex-align-center">
+              <div
+                className="grid-col-auto flex-align-center"
+                style={{
```

Comment:
Out of curiosity, why is this inline?

Reply:
No science to it tbh. I'll create a css class for this.